### DISCUSSION TITLE
Nuxt modules page

### DISCUSSION THREAD

**Discussion by rustem-nasyrov** [nuxt/nuxt#16094] [Questions] [answered]
Hello for everyone! Is there any way to customize the module page at the [nuxt modules](https://nuxt.com/modules)? Is it possible to display another readme not from root of the repository?

**Comment by atinux** [comment, marked as answer]
Could you create the issue on https://github.com/nuxt/modules ?

**Comment by rustem-nasyrov** [comment]
@Atinux, yeah, sure. Thank you.
